### ISSUE TITLE
Update page header for new links

### ISSUE THREAD

**Issue by rerb** [AASHE/hub#428]

**Comment by monika-aashe** [comment]
Need to have meeting.

**Comment by dodget** [comment]
This one is unclear to me. The links in the header appear to be valid and up to date.

**Comment by juliandsmith** [comment]
Yeah, this ticket isn't very clear. The idea here is that the header on the Hub should have the same look and feel as the headers on the AASHE and STARS sites.
